msal_extensions/libsecret.py

```diff
@@ -15,16 +15,32 @@
 """
 import logging
 
-import gi  # https://pygobject.readthedocs.io/en/latest/getting_started.html
-
-# pylint: disable=no-name-in-module
-gi.require_version("Secret", "1")  # Would require a package gir1.2-secret-1
-# pylint: disable=wrong-import-position
-from gi.repository import Secret  # Would require a package gir1.2-secret-1
-
-
 logger = logging.getLogger(__name__)
 
+try:
+    import gi  # https://pygobject.readthedocs.io/en/latest/getting_started.html
```

Comment:
If someone doesn't have logging configured, they won't see the explanation logged below, but this line will be part of the traceback. It could be better to link to an explanation in this library's documentation.

Reply:
Glad that you've also noticed that this file was deliberately arranged to have inline comment helpful in error trace.

I've also adopted your suggestion to link to a wiki page in this repo (which can be further refined at a later time).